Plugins/Public/base_plugin/Main.cpp

```diff
@@ -467,9 +478,28 @@ void LoadSettingsActual()
 					{
 						repair_per_repair_cycle = ini.get_value_int(0);
 					}
-					else if (ini.is_value("shield_damage_multiplier"))
+					else if (ini.is_value("shield_reinforcement_threshold_flat"))
+					{
+						shield_reinforcement_threshold_flat = ini.get_value_float(0);
+					}
+					else if (ini.is_value("shield_reinforcement_threshold_percent"))
+					{
+						shield_reinforcement_threshold_percent = ini.get_value_float(0);
+					}
+					else if (ini.is_value("shield_reinforcement_increment"))
 					{
-						set_shield_damage_multiplier = ini.get_value_float(0);
+						shield_reinforcement_increment = ini.get_value_float(0);
+					}
+					else if (ini.is_value("base_shield_strength"))
+					{
+						base_shield_strength = ini.get_value_float(0);
+					}
+					else if (ini.is_value("base_vulnerability_window"))
```

Comment:
I think only allowing hours here is a bit limiting.

Reply:
It is, but then we'd add another layer of complexity to the formula, after discussing with Haste, it was considered 'good enough'